lib/log_parser.rb

```diff
@@ -0,0 +1,16 @@
+class Log_Parser
```

Comment:
```suggestion
class LogParser
```

Nome de classes são declaradas em Pascal case
https://celsokitamura.com.br/pascal-case-e-camel-case/